packages/node/src/node-hid-wrapper.ts

```diff
@@ -18,11 +18,13 @@ export class NodeHIDDevice extends EventEmitter implements HIDDevice {
 	}
 
 	public write(data: number[]): void {
-		this.device.write(data)
+		this.device.write(data).catch((err) => {
+			this.emit('error', err)
+		})
```

Comment:
Just a question: Is it ok to write like this? ie we don't have to worry about concurrency or add some promise queue here?

Reply:
Hmm actually yeah, this does need to be put in a queue. 
`close` might need to be put in the same queue too.